sktime/classification/dictionary_based/_tde.py

```diff
@@ -269,33 +284,39 @@ def fit(self, X, y):
             tde.fit(X_subsample, y_subsample)
             tde.subsample = subsample
 
-            tde.accuracy = self._individual_train_acc(
+            if self.save_train_predictions:
+                tde._train_predictions = np.zeros(self.n_instances - subsample_size)
+
+            tde._accuracy = self._individual_train_acc(
                 tde,
                 y_subsample,
                 subsample_size,
                 -999999 if num_classifiers < self.max_ensemble_size else lowest_acc,
```

Comment:
numpy.NINF instead of -99999?

Reply:
Yeah, that makes more sense :)

Reply:
ah nvm it doesnt like `required_correct = int(lowest_acc * train_size)` with inf, im just going to go with 0 since its not as weird as whats there currently and works just as well.